### ISSUE TITLE
ZeroDivisionError: float division by zero when trying f_A = 0 with Gordon16_RvFALaw

### ISSUE THREAD

**Issue by galaxyumi** [BEAST-Fitting/beast#561]
While trying to get SMC-like extinction curves, I got an error for f_A = 0 cases. This happens when the code tries to figure out the Rv of the A component, which I want to ignore. For the SMC-like extinction curve, should I use the 'Gordon03_SMCBar' instead of 'Gordon16_RvFALaw'?

**Comment by karllark** [comment]
Yes, that will solve the problem.  But it would be good to have the `Gordon16_RvFALaw` work as expected with f_A = 0.  So, I'll make this issue as a bug and we should keep it open till it is fixed.
